config.js.docker

```diff
@@ -102,14 +102,21 @@ for (const x of seeds) {
 
   // Multisig configuration and validation
   const multisigPubkeys = getParam(`multisig_seed_${x}_pubkeys`);
-  const multisig = {
-    total:          getParam(`multisig_seed_${x}_max_signatures`),
-    numSignatures:  getParam(`multisig_seed_${x}_num_signatures`),
-  };
-  if (!(multisigPubkeys || multisig.total || multisig.numSignatures)) {
+  const multisigTotal = getParam(`multisig_seed_${x}_max_signatures`);
+  const multisigNumSigs = getParam(`multisig_seed_${x}_num_signatures`);
+  
+  if (!(multisigPubkeys || multisigTotal || multisigNumSigs)) {
     // No multisig parameters means this is not a multisig.
     continue;
   }
+
+  // This parse is necessary cause some infrastructures treat all env variables as string. 
+  // As the comparision is for a equal value (===) that would make it fail. 
+  const multisig = {
+    total:          parseInt(multisigTotal),
+    numSignatures:  parseInt(multisigNumSigs),
+  };
+
```

Comment:
I believe using `getIntParam` would be enough and more concise. Also, it has some additional protections against a strange parseInt behavior.

```suggestion
  const multisig = {
    total:          getIntParam(`multisig_seed_${x}_max_signatures`),
    numSignatures:  getIntParam(`multisig_seed_${x}_num_signatures`),
  };
  if (!(multisigPubkeys || multisig.total || multisig.numSignatures)) {
    // No multisig parameters means this is not a multisig.
    continue;
  }
```